### ISSUE TITLE
Version 1.3.6 failure: create_tag unrecognized

### ISSUE THREAD

**Issue by berkus** [pascalgn/npm-publish-action#36]
```
Unexpected input(s) 'create_tag', valid inputs are ['entryPoint', 'args', 'commit_pattern', 'tag_name', 'tag_message', 'workspace', 'publish_command', 'publish_args']
```

I used sample config yaml directly from marketplace page.

**Comment by dranidis** [comment]
Same here

**Comment by tiagoboeing** [comment]
Tags are created automatically. I don't use these options and the tags are created by this workflow.

See my workflow file: https://github.com/tiagoboeing/anywhere-webcomponents/blob/develop/.github/workflows/deploy.yml#L45

**Comment by pascalgn** [comment]
Sorry about this, the entry was missing in the action.yml file. It should be fixed now in the latest release (1.3.7) 👍
